### PR TITLE
Transfer endpoints of Weather Anomaly Data Service to this service

### LINKED ISSUE #71: Transfer endpoints of Weather Anomaly Data Service to this service
The [Weather Anomaly Data Service](https://github.com/pacificclimate/weather-anomaly-data-service) is the current backend for the [Weather Anomaly Viewer](https://github.com/pacificclimate/weather-anomaly-tool). 

The WADS accesses a PyCDS database and serves data extracted from the Weather Anomaly views and matviews. The endpoints code is simple. Its infrastructure is wildly out of date. Further, contemplated upgrades to the WAV will likely require metadata provided by the SDP backend. It makes sense to move the WADS endpoints here.

### PR DESCRIPTION
**Pull request by rod-glover**
Resolves #71 

TODO:

- [x] Transfer endpoints code
- [x] Add tests

### CODE REVIEW

**Comment by rod-glover** [comment]
Working locally

**Comment by rod-glover** [comment]
[Deployed](https://services.pacificclimate.org/dev/weather-anomaly-viewer/) to dev

**Review by corviday — APPROVED** [review]
Looks good to me. Straightforward code, good documentation, and extremely good testing. 

I didn't poke at the demo because unfortunately I'm not familiar enough with this system to construct test queries.

File: docs/api-design.md [hunk 1]
```diff
@@ -4,38 +4,43 @@
 
 We follow RESTful design principles in this microservice.
 
-We us the very common collection pattern in its
-RESTful expression, e.g., for the `/stations` endpoint.
+We use the very common collection pattern in its RESTful expression. The collection pattern is used when the underlying data can be naturally modelled as a list of individual objects. The list (collection) is a resource, and each individual network is a resource. Each has its own URI. 
 
-Overall design of API mainly reflects four core tables:
+For example, in CRMP, the most natural way to regard networks is as a collection of individual networks. It is hard to imagine a better way to represent these items. This is true for the majority of resources provided by this API.
```
[the remark this answers is not in the document]
Comment (rod-glover) [reply docs/api-design.md:9]: Thanks; terrible wording. Improved now.

**Pull request merged**